### ISSUE TITLE
`conda install` and `conda update` suggest a newer version of conda exists, but the actual update process reports all packages already installed

### ISSUE THREAD

**Issue by shadowwalkersb** [conda/conda#7236]
**I'm submitting a...**
  - [x] bug report
  - [ ] feature request

## Current Behavior

While running `conda install ...`, I got the warning saying that a newer version of `conda` is available. So, I tried to update `conda`. I got the same warning, but then `conda` reported that all packages are already installed. 😕 

It turns out a new version is, indeed, available on `conda-forge`, but not on `defaults`. So, the message is correct, but `conda update` doesn't want to update.

### Steps to Reproduce

```
$ conda update -n base conda
Solving environment: done


==> WARNING: A newer version of conda exists. <==
  current version: 4.5.1
  latest version: 4.5.2

Please update conda by running

    $ conda update -n base conda



# All requested packages already installed.

```

Specifying channel `defaults` gives the same result:
```
$ conda update -n base conda -c defaults
Solving environment: done


==> WARNING: A newer version of conda exists. <==
  current version: 4.5.1
  latest version: 4.5.2

Please update conda by running

    $ conda update -n base conda



# All requested packages already installed.

```

Specifying channel `conda-forge` works:
```
$ conda update -n base conda -c conda-forge
Solving environment: done

## Package Plan ##

  environment location: /Users/shadow_walker/Files/eclipse_workspace/workspace_work/minicondas/4.4.10

  added / updated specs:
    - conda


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    conda-4.5.2                |           py27_0         629 KB  conda-forge
    openssl-1.0.2o             |                0         3.3 MB  conda-forge
    ca-certificates-2018.4.16  |                0         140 KB  conda-forge
    certifi-2018.4.16          |           py27_0         142 KB  conda-forge
    ------------------------------------------------------------
                                           Total:         4.2 MB

The following packages will be UPDATED:

    ca-certificates: 2018.03.07-0      --> 2018.4.16-0      conda-forge
    certifi:         2018.4.16-py27_0  --> 2018.4.16-py27_0 conda-forge
    conda:           4.5.1-py27_0      --> 4.5.2-py27_0     conda-forge
    openssl:         1.0.2o-h26aff7b_0 --> 1.0.2o-0         conda-forge

Proceed ([y]/n)?
```

## Expected Behavior

`conda` should be updated as the output of `conda` suggests or the message could be more descriptive and report on which channel `conda` is available.

## Environment Information

<details open><summary><b>Probably relevant </b><code>`conda search conda`</code></summary><p>

```
$ conda search conda
Loading channels: done
# Name                  Version           Build  Channel
<Truncated>
conda                     4.5.1          py27_0  pkgs/main
conda                     4.5.1          py27_0  conda-forge
conda                     4.5.1          py35_0  pkgs/main
conda                     4.5.1          py35_0  conda-forge
conda                     4.5.1          py36_0  pkgs/main
conda                     4.5.1          py36_0  conda-forge
conda                     4.5.2          py27_0  conda-forge
conda                     4.5.2          py35_0  conda-forge
conda                     4.5.2          py36_0  conda-forge
```
</p></details>

<details open><summary><code>`conda info`</code></summary><p>

```
$ conda info

     active environment : None
            shell level : 0
       user config file : /Users/shadow_walker/.condarc
 populated config files : /Users/shadow_walker/.condarc
          conda version : 4.5.1
    conda-build version : 3.7.2
         python version : 2.7.14.final.0
       base environment : /Users/shadow_walker/Files/eclipse_workspace/workspace_work/minicondas/4.4.10  (writable)
           channel URLs : https://conda.anaconda.org/cryoem/osx-64
                          https://conda.anaconda.org/cryoem/noarch
                          https://repo.anaconda.com/pkgs/main/osx-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/osx-64
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/osx-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/pro/osx-64
                          https://repo.anaconda.com/pkgs/pro/noarch
                          https://conda.anaconda.org/conda-forge/osx-64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : /Users/shadow_walker/Files/eclipse_workspace/workspace_work/minicondas/4.4.10/pkgs
                          /Users/shadow_walker/.conda/pkgs
       envs directories : /Users/shadow_walker/Files/eclipse_workspace/workspace_work/minicondas/4.4.10/envs
                          /Users/shadow_walker/.conda/envs
               platform : osx-64
             user-agent : conda/4.5.1 requests/2.18.4 CPython/2.7.14 Darwin/15.6.0 OSX/10.11.6
                UID:GID : 501:20
             netrc file : None
           offline mode : False

```
</p></details>

<details open><summary><code>`conda config --show-sources`</code></summary><p>

```
$ conda config --show-sources
==> /Users/shadow_walker/.condarc <==
ssl_verify: True
channels:
  - cryoem
  - defaults
  - conda-forge

```
</p></details>

<details><summary><code>`conda list --show-channel-urls`</code></summary><p>

```
$ conda list --show-channel-urls
# packages in environment at /Users/shadow_walker/Files/eclipse_workspace/workspace_work/minicondas/4.4.10:
#
# Name                    Version                   Build  Channel
appnope                   0.1.0            py27hb466136_0    defaults
asn1crypto                0.24.0                   py27_0    defaults
backports                 1.0              py27hb4f9756_1    defaults
backports.functools_lru_cache 1.5                      py27_1    defaults
backports.shutil_get_terminal_size 1.0.0            py27hc9115de_2    defaults
backports_abc             0.5              py27h6972548_0    defaults
beautifulsoup4            4.6.0            py27h9416283_1    defaults
boost                     1.63.0                   py27_5    conda-forge
boost-cpp                 1.63.0                        2    conda-forge
bsddb                     1.0                      py27_1    defaults
bzip2                     1.0.6                hd86a083_4    defaults
ca-certificates           2018.03.07                    0    defaults
cctools                   895                  h7512d6f_0    defaults
certifi                   2018.4.16                py27_0    defaults
cffi                      1.11.4           py27h342bebf_0    defaults
chardet                   3.0.4            py27h2842e91_1    defaults
clang                     4.0.1                h662ec87_0    defaults
clang_osx-64              4.0.1                h1ce6c1d_8    defaults
clangxx                   4.0.1                hc9b4283_0    defaults
clangxx_osx-64            4.0.1                h22b1bf0_8    defaults
cmake                     3.9.4                h30c3106_0    defaults
compiler-rt               4.0.1                h5487866_0    defaults
conda                     4.5.1                    py27_0    defaults
conda-build               3.7.2                    py27_0    defaults
conda-env                 2.6.0                h36134e3_0    defaults
conda-verify              2.0.0            py27hbebe81d_0    defaults
contextlib2               0.5.5            py27h9cb85f4_0    defaults
cryptography              2.1.4            py27hdbc5e8f_0    defaults
curl                      7.59.0               ha441bb4_0    defaults
cycler                    0.10.0           py27hfc73c78_0    defaults
db                        5.3.28                        1    defaults
decorator                 4.2.1                    py27_0    defaults
eman-deps                 8.0                 np113py27_0    cryoem
enum34                    1.1.6            py27hf475452_1    defaults
expat                     2.2.5                hb8e80ba_0    defaults
fftw-mpi                  3.3.6                         2    cryoem
filelock                  3.0.4                    py27_0    defaults
freetype                  2.8                  h12048fb_1    defaults
ftgl                      2.1.3                         1    cryoem
functools32               3.2.3.2          py27h8ceab06_1    defaults
futures                   3.2.0            py27h1b80678_0    defaults
glob2                     0.6              py27h55c9705_0    defaults
gsl                       2.2.1                h002c638_3    defaults
hdf5                      1.8.18               h017327d_1    defaults
icu                       58.2                 h4b95b61_1    defaults
idna                      2.6              py27hedea723_1    defaults
intel-openmp              2018.0.0             h8158457_8    defaults
ipaddress                 1.0.19                   py27_0    defaults
ipython                   5.4.1                    py27_2    defaults
ipython_genutils          0.2.0            py27h8b9a179_0    defaults
jinja2                    2.10             py27h70b8dc5_0    defaults
jpeg                      9b                   he5867d9_2    defaults
ld64                      274.2                h7c2db76_0    defaults
libcurl                   7.59.0               hf30b1f0_0    defaults
libcxx                    4.0.1                h579ed51_0    defaults
libcxxabi                 4.0.1                hebd6815_0    defaults
libedit                   3.1                  hb4e282d_0    defaults
libffi                    3.2.1                h475c297_4    defaults
libgfortran               3.0.1                h93005f0_2    defaults
libpng                    1.6.34               he12f830_0    defaults
libssh2                   1.8.0                h322a93b_4    defaults
libtiff                   4.0.9                h0dac147_0    defaults
libuv                     1.19.2               hfd94794_0    defaults
llvm                      4.0.1                hc748206_0    defaults
llvm-lto-tapi             4.0.1                h6701bc3_0    defaults
markupsafe                1.0              py27hd3c86fa_1    defaults
matplotlib                2.1.2            py27h6d6146d_0    defaults
mkl                       2018.0.1             hfbd8650_4    defaults
mkl-service               1.1.2            py27hcba3029_4    defaults
ncurses                   6.0                  hd04f020_2    defaults
nose                      1.3.7            py27h2ee3cb8_2    defaults
numpy                     1.13.3           py27h8a80b8c_2    defaults
openmpi                   2.0.2                         0    cryoem
openssl                   1.0.2o               h26aff7b_0    defaults
pathlib2                  2.3.0            py27he09da1e_0    defaults
pexpect                   4.4.0                    py27_0    defaults
pickleshare               0.7.4            py27h37e3d41_0    defaults
pip                       9.0.1            py27h1567d89_4    defaults
pkginfo                   1.4.1            py27ha9221e7_0    defaults
prompt_toolkit            1.0.15           py27h4a7b9c2_0    defaults
psutil                    5.4.3            py27h1de35cc_0    defaults
ptyprocess                0.5.2            py27h70f6364_0    defaults
pycosat                   0.6.3            py27h6c51c7e_0    defaults
pycparser                 2.18             py27h0d28d88_1    defaults
pydusa                    1.15                    np113_9    cryoem
pygments                  2.2.0            py27h1a556bb_0    defaults
pyopengl                  3.1.0                    py27_0    conda-forge
pyopenssl                 17.5.0           py27hfda213f_0    defaults
pyparsing                 2.2.0            py27h5bb6aaf_0    defaults
pyqt                      4.11.4                   py27_4    defaults
pysocks                   1.6.7            py27h1cff6a6_1    defaults
python                    2.7.14              hde5916a_29    defaults
python-dateutil           2.6.1            py27hd56c96b_1    defaults
python.app                2                py27hf2d5e94_7    defaults
pytz                      2018.3                   py27_0    defaults
pyyaml                    3.12             py27ha7932d0_1    defaults
qt                        4.8.7                         4    defaults
readline                  7.0                  hc1231fa_4    defaults
requests                  2.18.4           py27h9b2b37c_1    defaults
rhash                     1.3.5                h3aa0507_1    defaults
ruamel.yaml               0.15.37                   <pip>
ruamel_yaml               0.15.35          py27h1de35cc_1    defaults
scandir                   1.7              py27h1de35cc_0    defaults
scikit-learn              0.19.1           py27h9788993_0    defaults
scipy                     1.0.0            py27h793f721_0    defaults
setuptools                38.4.0                   py27_0    defaults
simplegeneric             0.8.1                    py27_2    defaults
singledispatch            3.4.0.3          py27he22c18d_0    defaults
sip                       4.18                     py27_0    defaults
six                       1.11.0           py27h7252ba3_1    defaults
sqlite                    3.22.0               h3efe00b_0    defaults
ssl_match_hostname        3.5.0.1          py27h8780752_2    defaults
subprocess32              3.2.7            py27h24b2887_0    defaults
theano                    1.0.1            py27h04f5b5a_0    defaults
tk                        8.6.7                h35a86e2_3    defaults
tornado                   4.5.3                    py27_0    defaults
traitlets                 4.3.2            py27hcf08151_0    defaults
urllib3                   1.22             py27hc3787e9_0    defaults
wcwidth                   0.1.7            py27h817c265_0    defaults
wheel                     0.30.0           py27h677a027_1    defaults
xz                        5.2.3                h0278029_2    defaults
yaml                      0.1.7                hc338f04_2    defaults
zlib                      1.2.11               hf3cbc9b_2    defaults
```
</p></details>

**Comment by kalefranz** [comment]
Thanks for the report.  This is a duplicate of https://github.com/conda/conda/issues/7165.

**Comment by mbargull** [comment]
https://github.com/conda/conda/issues/7165#issuecomment-389320976
> Resolved with #7289 to be released in 4.5.5.

**Comment by Mark531** [comment]
I use conda 4.5.11 and I still have the problem:
==> WARNING: A newer version of conda exists. <==
  current version: 4.5.11
  latest version: 4.6.4

Please update conda by running

    $ conda update -n base -c defaults conda

**Comment by RileyMShea** [comment]
I also still have this problem

```
`C:\WINDOWS\system32>conda update -n base -c defaults conda
Solving environment: done


==> WARNING: A newer version of conda exists. <==
  current version: 4.5.11
  latest version: 4.6.7

Please update conda by running

    $ conda update -n base -c defaults conda



# All requested packages already installed.`
```

**Comment by jjhelmus** [comment]
@Mark531 @RileyMShea What versions of Python are in your base environment?  Conda 4.6 requires at Python 2.7 or 3.6, or 3.7.  If you have Python 3.5 you will need to upgrade Python prior to upgrading conda.

**Comment by Mark531** [comment]
@jjhelmus I use python 3.5.4 so it may explain the problem. But how come that this requirement doesn't appear when calling "conda update"? Ideally, conda should warn the user that a more recent version of python is required, and even install it if the user clicks on "yes".
@kalefranz you closed this issue, but it persists in 4.5.11 version of conda. So, for me, it's not fixed.

**Comment by mingwandroid** [comment]
@Mark531 

> I use python 3.5.4

> @kalefranz you closed this issue, but it persists in 4.5.11 version of conda. So, for me, it's not fixed.

We no longer support Python 3.5. Please update. We cannot support a build matrix that includes such very old python versions (2.7.x is a special case as it's quite a different language);\

So for us, the problem is fixed.

**Comment by Mark531** [comment]
Ok, I tried to update to python 3.6 with conda, and I got an error:

> C:\>conda install python=3.6 anaconda=custom
> Solving environment: failed
> 
> UnsatisfiableError: The following specifications were found to be in conflict:
>   - ipython-qtconsole -> pyqt=4 -> python=3.5
>   - python=3.6
> Use "conda info <package>" to see the dependencies for each package.

Why can't the library qtconsole be updated to 3.6? Is it used by Spyder?
